src/features/sprints/components/sprint-card.tsx

```
@@ -0,0 +1,95 @@
+    const handleStatusChange = async (status: "planning" | "active" | "completed" | "cancelled") => {
+        try { await updateSprint({ sprintId: sprint._id, status }); toast.success(`Sprint marked as ${STATUS_CONFIG[status].label}`); }
+        catch { toast.error("Failed to update sprint"); }
+    };
+
+    const handleDelete = async () => {
+        try { await removeSprint({ sprintId: sprint._id }); toast.success("Sprint deleted"); }
+        catch { toast.error("Failed to delete sprint"); }
+    };
+
+    return (
+        <div className={cn("group rounded-lg border bg-white p-4 shadow-sm transition-shadow hover:shadow-md cursor-pointer", sprint.status === "active" && "border-emerald-200 ring-1 ring-emerald-100")} onClick={onClick}>
```

Biome (2.4.15)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)